oak_functions/lookup_data_generator/src/data.rs

```diff
@@ -98,18 +100,50 @@ pub fn generate_and_serialize_sparse_weather_entries<R: Rng>(
     entries: usize,
 ) -> anyhow::Result<BytesMut> {
     let mut buf = BytesMut::new();
-    let lat_dist = rand::distributions::Uniform::new(-90_000, 90_000);
-    let lon_dist = rand::distributions::Uniform::new(-180_000, 180_000);
+    let pi = std::f32::consts::PI;
```

Comment:
Do the deploy scripts automatically populate the storage with freshly generated data or do we have to manually generate and upload the data to be able to use this version?

Reply:
No, but I already published a new version of the lookup data that will work with the old linear scanning code and this new code to the cloud storage.